src/Interactive/HostTest/InteractiveHostTest.csproj

```diff
@@ -10,7 +10,7 @@
     <RootNamespace>Roslyn.InteractiveHost.UnitTests</RootNamespace>
     <AssemblyName>Roslyn.InteractiveHost.UnitTests</AssemblyName>
     <TargetFramework>net461</TargetFramework>
-    <RuntimeIdentifier>win-x86</RuntimeIdentifier>
+    <RuntimeIdentifier>$(RoslynDesktopRuntimeIdentifierX86)</RuntimeIdentifier>
```

Comment:
Ah. Well that solves the mystery of why we were keeping those x86 monikers around.